pages/common/chmod.md

```diff
@@ -22,6 +22,10 @@
 
 `chmod o=g {{file}}`
 
+- Remove all rights from others:
```

Comment:
```suggestion
- Remove all rights from [o]thers:
```

Reply:
I didn't include the brackets here because it seemed that after showing it for one set or for one right, it didn't mark them again, e.g. using `[u]ser` on line 5 but `user` on line 9. I've committed changes to always include the brackets throughout the page.